### DISCUSSION TITLE
erreur dans la soumission 77

### DISCUSSION THREAD

**Discussion by jfcouchot** [danymat/INSAnonym-utils#73] [Q&A, Bug report & Support]
Rebonjour.

#77 FemtOrange2 | -1 | -1 | Status : Please report the error to the administrator, by sharing the file name.
-- | -- | -- | --
le fichier est 

FemtOrange.zip qui contient le csv : df_final_DEL_t.csv
Merci pour votre aide.
JF

**Comment by danymat** [comment]
Rebonjour, 

Il est obligatoire de garder le même format dans le fichier (c'est un peu une contrainte de cette compétition).
Donc les lignes où il y a des valeurs vides (longitude et latitude) dans votre cas, vont poser problème. Elles doivent rester en format float

**Reply by jfcouchot** [reply]
Merci Puis-je donc mettre sur chaque ligne 
DEL\tTIME\t0.0\t0.0

**Reply by danymat** [reply]
Si vous mettez DEL, faites attention à ne pas changer le reste de la ligne
